automation_tools/__init__.py

```diff
@@ -558,6 +558,19 @@ def install_puppet_scap_client():
 
 def setup_foreman_discovery():
     """Task to setup foreman discovery."""
+    # Install required packages to enable discovery plugin
+    packages = [
+        'tfm-rubygem-foreman_discovery'
+        'rubygem-smart_proxy_discovery'
+        'foreman-discovery-image'
+        'tfm-rubygem-hammer_cli_foreman_discovery'
+    ]
+    for package in packages:
```

Comment:
By the way, you could get them all installed in one shot (and not use a loop) by doing something like:

```python
run('yum install -y {0}'.format(" ".join(package)))
```